pkg/service/healthcheck/service.go

```diff
@@ -418,10 +419,12 @@ func (s *Service) nodeInfo(ctx context.Context, clusterID uuid.UUID, host string
 		}
 		if tlsEnabled {
 			tlsConfig, err := s.tlsConfig(clusterID, clientCertAuth)
-			if err != nil {
+			if err != nil && !errors.Is(err, service.ErrNotFound) {
 				return ni, errors.Wrap(err, "fetch TLS config")
 			}
-			ni.TLSConfig[p] = tlsConfig
+			if !errors.Is(err, service.ErrNotFound) {
+				ni.TLSConfig[p] = tlsConfig
+			}
 		}
```

Comment:
The logic becomes bit fuzzy now.
It checks `nodeInfo` response if the TLS is enabled, 
if it is and if it supposed to be protected with the certificate, try to get the certificate
if the certificate is not there (we don't store it), then finally fallback to non TLS connection.

Previously service.ErrNotFound was wrapped with 
```
			return nil, errors.Wrap(err, "get SSL user cert from secrets store")
```
, now this information will be ignored.

I can imagine the situation where the TLS is enabled, there is no way to connect without TLS, but someone forgot to add the cert. SM will create the session, but the end-user will have some information like "cannot connect to the cluster" or whatever. No information about missing SSL cert.

Please don't ignore ErrNotFound, log it with Info() level adding that it will fallback to TLS disabled connection.

Reply:
It may be that just logging is not a good idea.
Healtcheck is executed often, so it means that there is chance to SPAM logs.

Is there a chance to check if nonSSL is enabled ?

Reply:
As discussed offline, it's not possible to determine whether given port is open from Scylla API (https://github.com/scylladb/scylladb/issues/7206#issuecomment-689598343).
Added logging for case when TLS is enabled in config, but Manager is missing certificate to make it better visible to the user.